CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented here.
 
+## Version [4.7.2]
+
+- Improvement: update error messages for both whitelist authenticators
```

Comment:
Improvements increase the second part (minor) of a semantic version: https://semver.org/

So version would increase to `4.8.0` here :slightly_smiling_face: